index.html

```diff
@@ -192,514 +225,421 @@ <h2 class="sign">
-                <h1>Prof. Sridhar Hannenhalli</h1>
-                <br />
-                <p>
-                  <br />Senior Investigator, Cancer Data Science Laboratory,
-                  Center for Cancer Research, National Cancer Institute, MD, USA
-                </p>
-              </div>
-            </div>
-            <div class="judges img right">
-              <img src="assets/speaker6.jpg" />
-              <div class="team-content">
-                <h1>Prof. Pravindra Kumar</h1>
-                <br />
-                <p>
-                  <br />Professor, Department of Biotechnology, Indian Institute
-                  of Technology, Roorkee, India
-                </p>
-              </div>
-            </div>
-            <div class="judges img">
-              <img src="assets/speaker7.jpg" />
-              <div class="team-content">
-                <h1>Prof. T. Pullaiah</h1>
-                <br />
-                <p>
-                  <br />Professor of Botany (Retd.), UGC BSR Faculty Fellow,
-                  Department of Botany, Sir Krishnadevaraya University,
-                  Anantpur, India
-                </p>
-              </div>
-            </div>
-            <div class="judges img right">
-              <img src="assets/speaker8.jpg" />
-              <div class="team-content">
-                <h1>Prof. Nilesh Kumar Sharma</h1>
-                <br />
-                <p>
-                  Professor (Cancer Biology and Medical Biotechnology), Dr. D.
-                  Y. Patil Vidyapeeth, Pune, India
-                </p>
-              </div>
-            </div>
-            <div class="judges img">
-              <img src="assets/speaker9.jpg" />
-              <div class="team-content">
-                <h1>Dr. Gopal B Krishnan</h1>
-                <br />
-                <p>
-                  <br />Senior Global Product Manager-Biologics, Promega
-                  Corporations- USA
-                </p>
-              </div>
-            </div>
-            <div class="judges img right">
-              <img src="assets/speaker10.jpg" />
-              <div class="team-content">
-                <h1>Dr. Pavan Jutur</h1>
-                <br />
-                <p>
-                  <br />Group Leader, Omics of Algae, Industrial Biotechnology,
-                  International Centre for Genetic Engineering and Biotechnology
-                  (ICGEB), New Delhi
-                </p>
-              </div>
-            </div>
-            <div class="judges img">
-              <img src="assets/speaker11.jpg" />
-              <div class="team-content">
-                <h1>Dr. Mahesh Babu</h1>
-                <br />
-                <p>
-                  Director, IL&FS Environmental Infrastructure & Services Ltd.
-                  Delhi
-                </p>
-              </div>
-            </div>
-          </div>
-        </div>
+
+      <div class="title">ABOUT</div>
```

Comment:
Use classes like main and container, read css for them, they help to style things correctly

Reply:
Thanks sir!